packages/sdk/src/types/runtime/actor.ts

```diff
@@ -311,59 +314,67 @@ export class Actor implements ActorLike, Patchable<ActorLike> {
-    //     isTrigger: boolean,
-    //     center?: Vector3Like,
-    //    radius?: number): ForwardPromise<Collider>;
-
-    // /**
-    //  * Adds a collider of the given type and parameters on the actor.
-    //  * @param colliderType Type of the collider to enable.
-    //  * @param collisionLayer The layer that the collider operates in.
-    //  * @param isTrigger Whether the collider is a trigger volume or not.
-    //  * @param center The center of the collider, or default of the object if none is provided.
-    //  * @param size
-    //  */
-    // public enableCollider(
-    //     colliderType: 'box',
-    //     collisionLayer: CollisionLayer,
-    //     isTrigger: boolean,
-    //     center?: Vector3Like,
-    //     size?: Vector3Like): ForwardPromise<Collider>;
-
-    // /** @ignore */
-    // public enableCollider(
-    //     colliderType: 'box' | 'sphere',
-    //     collisionLayer: CollisionLayer,
-    //     isTrigger: boolean,
-    //     center?: Vector3Like,
-    //     size?: number | Vector3Like
-    // ): ForwardPromise<Collider> {
-    //     if (!this._collider) {
-    //         this._collider = new Collider(this);
-    //         observe(this._collider, 'collider', (...path: string[]) => this.actorChanged(...path));
-    //         this.subscribe('collider');
-    //         return this.context.internal.enableCollider(
-    //             this.id,
-    //             colliderType,
-    //             collisionLayer,
-    //             isTrigger,
-    //             center,
-    //             size);
-    //     }
-    //     return ForwardPromise.Resolve(this._collider);
-    // }
+    /**
+     * Adds a collider of the given type and parameters on the actor.
+     * @param colliderType Type of the collider to enable.
+     * @param collisionLayer The layer that the collider operates in.
+     * @param isTrigger Whether the collider is a trigger volume or not.
+     * @param center The center of the collider, or default of the object if none is provided.
+     * @param radius The radius of the collider, or default bounding if non is provided.
+     */
+    public enableCollider(
+        colliderType: 'sphere',
+        collisionLayer: CollisionLayer,
+        isTrigger: boolean,
+        center?: Vector3Like,
+        radius?: number): void;
+
+    /**
+     * Adds a collider of the given type and parameters on the actor.
+     * @param colliderType Type of the collider to enable.
+     * @param collisionLayer The layer that the collider operates in.
+     * @param isTrigger Whether the collider is a trigger volume or not.
+     * @param center The center of the collider, or default of the object if none is provided.
+     * @param size
+     */
+    public enableCollider(
+        colliderType: 'box',
+        collisionLayer: CollisionLayer,
+        isTrigger: boolean,
+        center?: Vector3Like,
+        size?: Vector3Like): void;
+
+    /** @ignore */
+    public enableCollider(

```

Comment:
It could be useful to store the collider options somewhere in the actor for retrieval later. Not sure how frequently it'll be useful, but seems like good practice.

Reply:
Not sure what options you are referring to.  The collider params?

Reply:
Right, the collider params. Since collider settings are immutable, it's conceivable that a user might pool and reuse collider actors. Having after-the-fact access to the config would be useful for this case.